pkg/clustersync/clusterSync.go

```diff
@@ -20,8 +39,228 @@ func ElectLeaderAndStart(ctx context.Context) {
 }
 
 func syncClusters(ctx context.Context) {
-	klog.Info("TODO: Start Sync clusters here.")
+	for {
+		select {
+		case <-ctx.Done():
+			klog.Info("Exit syncClusters.")
+			return
+		default:
+			klog.V(1).Info("Attempting to sync clusters.")
+			WatchClusters(ctx)
+		}
+	}
+}
```

Comment:
This will keep calling WatchClusters() in a loop.
We only need this:
```go
WatchClusters(ctx)
<-ctx.Done():
klog.Info("Exit syncClusters.") // I don't think we need to wait just to log this message. The context should take care of stopping the child routines.
```

Reply:
Done